docs/wiki/User Guide/REST-APIs.md

```diff
@@ -231,7 +231,7 @@ Supported parameters are:
 
 User can use `user_task_ids`/`client_ids`/`endpoints`/`types` make Cruise Control only return requests they are interested. By default all the requests get returned.
 
-If `fetch_completed_task` is set to `true`, the original response of each requests will be returned.
+If `fetch_completed_task` is set to `true`, the original response of each requests will be returned. In the case where a task completed with errors the response will be `"CompletedWithError"`.
```

Comment:
Nit: Double quotes (i.e. `"`) seems redundant since `CompletedWithError ` is already inside backticks.

Reply:
Good point.